packages/eas-cli/src/project/projectUtils.ts

```diff
@@ -31,6 +36,10 @@ export function getProjectAccountName(exp: ExpoConfig, user: Actor): string {
   }
 }
 
+/**
+ * @deprecated - prefer using the account that definitively owns the project by
+ *               fetching it via the App.ownerAccount GraphQL field.
+ */
```

Comment:
don't deprecate this (it's for just getting the logged-in user's username). Remove dependence on ExpoConfig though by removing the arg.

Reply:
https://github.com/expo/eas-cli/pull/1370